data/css/modules/card/_list.scss

```diff
@@ -36,27 +34,42 @@
 
     &__content_grid {
         padding: 20px;
+        transition: padding 150ms ease-in-out;
     }
 
-    &:hover &__title {
-        color: $primary-light-color;
-    }
-
-    &.next:hover &__title {
-        color: $primary-medium-color;
+    &:hover {
+        box-shadow: 0px 4px 5px transparentize(black, 1 - 0.35);
+        .CardList__title {
+            color: $accent-light-color;
+        }
     }
 
-
     &.width-e &__title {
         font-size: 18px;
     }
 
-    &.width-f &__title {
-        font-size: 23px;
+    &.width-f {
+        &__title {
```

Comment:
```
**&.width-f &__title **
```

and 

```
&.width-f {
         &__title {
```

Does not generate the same thing, and its actually generating something wrong.